kotlin-eclipse-ui/src/org/jetbrains/kotlin/ui/builder/KotlinBuilder.kt

```diff
@@ -107,6 +115,19 @@ class KotlinBuilder : IncrementalProjectBuilder() {
         return null
     }
 
+    override fun startupOnInitialize() {
```

Comment:
Why exactly here?
Maybe it's worth moving this code to [`KotlinNature`](https://github.com/JetBrains/kotlin-eclipse/blob/master/kotlin-eclipse-core/src/org/jetbrains/kotlin/core/model/KotlinNature.kt#L65)? Plus, it would nice to remove this filter for Kotlin files in deconfiguration phase.

Reply:
I was thinking about moving it to `KotlinNature.addNature`, so that it will be called only once per project, but then it would not change already existing projects. As for the cons of the approach you suggested, we would then need to remember previous value of this parameter.

Reply:
Aha, I see, thank you. Then let's stick to your approach